packages/bazel-bot/docker-image/docker-main.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+git clone https://github.com/googleapis/googleapis.git
+git clone git@github.com:googleapis/googleapis-gen.git
+
+export GOOGLEAPIS=googleapis
+export GOOGLEAPIS_GEN=googleapis-gen
+export BAZEL_REMOTE_CACHE=https://storage.googleapis.com/repo-automation-bots-bazel-cache
+
+# TODO: install git credentials.
```

Comment:
Since the container runs longer than 10 minutes, I think we will want to pass in a GitHub app's private key as an argument, which would then be used to create a short lived (10 minute) JWT, to perform operations on GitHub as needed.

I think we shouldn't build this into the container, as it would make leaking the container a security risk.

Reply:
Yep.  Let's do it in the next pull request.